.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 *.log
 site
 coverage
+jsconfig.json
```

Comment:
https://code.visualstudio.com/docs/languages/jsconfig

It should be in .gitignore.